DigitalLearningSolutions.Web/Controllers/TrackingSystem/CourseSetup/AdminFieldsController.cs

```diff
@@ -222,15 +313,62 @@ private void SetEditAdminFieldTempData(EditAdminFieldViewModel model)
             TempData.Set(data);
         }
 
+        private IActionResult AddAdminFieldPostSave(AddAdminFieldViewModel model)
+        {
+            ModelState.ClearAllErrors();

```

Comment:
Not sure actually, I copied it from how the Registration Prompts were set up. @AlexJacksonDS is this necessary here?

Reply:
Ah Alex nevermind, I understand how ModelState.ClearAllErrors() works now!

Reply:
I've changed it to clear all errors except the AdminFieldId so that only the Answer field errors are cleared before we check if the model state is valid

Reply:
And I've added some extra unit tests to check these conditions :)